types/aws-lambda/index.d.ts

```diff
@@ -322,6 +323,15 @@ export interface CognitoUserPoolTriggerEvent {
         messageAction?: "SUPPRESS";
         desiredDeliveryMediums?: Array<"EMAIL" | "SMS">;
         forceAliasCreation?: boolean;
+        claimsOverrideDetails?: {
+            claimsToAddOrOverride?: { [key: string]: string };
+            claimsToSuppress?: string[];
+            groupOverrideDetails?: {
```

Comment:
Docs has "If you provide an empty or null object in the response, then the groups are suppressed.", so maybe allowing `null` for this?

Reply:
Good point, fixed.